libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -123,4 +93,29 @@ private void muteTts() {
       textToSpeech.stop();
     }
   }
+
+  public void setInstructionListener(final InstructionListener instructionListener) {
```

Comment:
> Access can be package-private

Reply:
get/set nomenclature is hijacked by the Java language and should be only used when defining POJOs/DTOs.
Could we rename the method differently?

Reply:
I didn't like that it was previously `addInstructionListener`, because that implies that you can have multiple. This is pretty common way to add listeners, what would you suggest?

Reply:
`establish`? `assign`? `put`?